python/lsst/afw/math/_boundedField.cc

```diff
@@ -57,42 +58,43 @@ void declareTemplates(PyClass &cls) {
     cls.def("divideImage", &BoundedField::divideImage<PixelT>,
             "image"_a, "overlapOnly"_a = false, "xStep"_a = 1, "yStep"_a = 1);
 }
-
-PYBIND11_MODULE(boundedField, mod) {
-    PyClass cls(mod, "BoundedField");
-
-    table::io::python::addPersistableMethods<BoundedField>(cls);
-
-    cls.def("__rmul__", [](BoundedField &bf, double const scale) { return bf * scale; }, py::is_operator());
-    cls.def("__mul__", &BoundedField::operator*, py::is_operator());
-    cls.def("__truediv__", &BoundedField::operator/, py::is_operator());
-    cls.def("__eq__", &BoundedField::operator==, py::is_operator());
-    cls.def("__ne__", &BoundedField::operator!=, py::is_operator());
-
-    cls.def("evaluate", (double (BoundedField::*)(double, double) const) & BoundedField::evaluate);
-    cls.def("evaluate",
-            (ndarray::Array<double, 1, 1> (BoundedField::*)(ndarray::Array<double const, 1> const &,
-                                                            ndarray::Array<double const, 1> const &) const) &
-                    BoundedField::evaluate);
-    cls.def("evaluate",
-            (double (BoundedField::*)(lsst::geom::Point2D const &) const) & BoundedField::evaluate);
-    cls.def("integrate", &BoundedField::integrate);
-    cls.def("mean", &BoundedField::mean);
-    cls.def("getBBox", &BoundedField::getBBox);
-
-    // Pybind11 resolves overloads by picking the first one that might work
-    declareTemplates<double>(cls);
-    declareTemplates<float>(cls);
-
-    utils::python::addOutputOp(cls, "__str__");
-    cls.def("__repr__", [](BoundedField const &self) {
-        std::ostringstream os;
-        os << "BoundedField(" << self << ")";
-        return os.str();
-    });
+} //namespace
+void wrapBoundedField(lsst::utils::python::WrapperCollection &wrappers) {
+
```

Comment:
Needs `addSignatureDependency("lsst.afw.table.io")` to support `addPersistableMethods`.